packages/fetchai/skills/weather_station/README.md

```diff
@@ -0,0 +1,36 @@
+# Weather Station
+
+## Description
+
+This skill sells dummy weather data.
+
+This skill is part of a Fetch.ai weather demo. It reads data from a database, that is populated with  dummy data from a weather station. It can be requested (for example by an agent with the weather_client skill) to provide weather data for specific dates, which it delivers after it receives payment.
+
+## Behaviours
+
+* service_registration: registers service on soef search service 
+
+## Handlers
+
+* fipa: handles fipa messages for negotiation
+* ledger_api: handles ledger_api messages for payment
+* oef_search: handles oef_search messages if service registration on soef is unsuccessful
+
+## Dependencies
```

Comment:
It has dependencies; but I think this is not needed as it duplicates information from the specification which then means we have to maintain it in two places. So I'd remove these Dependencies here